packages/core/test/unit/clients/imomentocache-interface.test.ts

```diff
@@ -0,0 +1,34 @@
+import {Project} from 'ts-morph';
+
+describe('IMomentoCache', () => {
+  it('has all of the data plane methods that exist on the ICacheClient interface', () => {
```

Comment:
Nice, thank you!

Reply:
this test actually failed when I rebased because Rishti had added `dictionaryLength`. So it was a good call.